eZ/Publish/Core/Persistence/Cache/TrashHandler.php

```diff
@@ -99,7 +99,25 @@ public function findTrashItems(Criterion $criterion = null, $offset = 0, $limit
     public function emptyTrash()
     {
         $this->logger->logCall(__METHOD__, array());
+
+        $trashedItems = $this->persistenceHandler->trashHandler()->findTrashItems();
+
+        $tags = [[]];
+        foreach ($trashedItems as $trashedItem) {
+            $contentId = $trashedItem->contentId;
+            $reverseRelations = $this->persistenceHandler->contentHandler()->loadReverseRelations($contentId);
+
+            if (!empty($reverseRelations)) {
```

Comment:
given it always returns array we can skip this if right?
But we should probably move it to `invalidateTags()` so we don't call that with empty tags array.

Reply:
Thanks @andrerom, it is fixed now